pyinjective/async_client.py

```diff
@@ -73,6 +78,8 @@ def __init__(
         self.stubInsurance = insurance_rpc_grpc.InjectiveInsuranceRPCStub(self.exchange_channel)
         self.stubSpotExchange = spot_exchange_rpc_grpc.InjectiveSpotExchangeRPCStub(self.exchange_channel)
         self.stubDerivativeExchange = derivative_exchange_rpc_grpc.InjectiveDerivativeExchangeRPCStub(self.exchange_channel)
+        self.stubExplorer = explorer_rpc_grpc.InjectiveExplorerRPCStub(exchange_channel)
+        self.stubAuction = auction_rpc_grpc.InjectiveAuctionRPCStub(exchange_channel)
```

Comment:
should be self.exchange_channel

Reply:
Cool, I'll change didn't notice the refactoring.